MVP/defaults-build.yaml

```diff
@@ -1,6 +1,23 @@
 - defaults:
     name: defaults-build
     node: 'jslave-platform-rhel7'
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag
```

Comment:
$tag will be empty for scratch builds (also $task_id - since scratch build task id will be in $id).
Should be fine to leave this field empty while triggering a job with scratch build info manually.